src/popup/router/components/NodeConnectionStatus.vue

```diff
@@ -49,7 +52,6 @@ export default {
     position: sticky;
   }
 
-  bottom: 48px;
```

Comment:
We have that backup property to support old browser versions.
Let's consider dropping of support for the old browsers in other PR. Or at least investigate it. If you want you can create an issue to think about it.